dragonbane.js

```diff
@@ -338,20 +345,35 @@ Hooks.once('diceSoNiceReady', (dice3d) => {
             "systems/dragonbane/art/ui/dsn/dod-ikon-demon-vit-256.png"
         ],
         bumpMaps: [
-        "systems/dragonbane/art/ui/dsn/dod-ikon-drake-bump.png",
-        ,,,,,,,,,,,,,,,,,,
-        "systems/dragonbane/art/ui/dsn/dod-ikon-demon-bump.png"
+            "systems/dragonbane/art/ui/dsn/dod-ikon-drake-bump.png",
+            , , , , , , , , , , , , , , , , , ,
+            "systems/dragonbane/art/ui/dsn/dod-ikon-demon-bump.png"
         ],
         system: 'dragonbane',
     });
 });
 
+// Set up Year Zero Engine Combat
+Hooks.on("yzeCombatReady", () => {
+    if (game.settings.get("dragonbane", "configuredYzeCombat")) return;
+    try {
+        game.settings.set("yze-combat", "resetEachRound", true);
+        game.settings.set("yze-combat", "slowAndFastActions", false);
+        game.settings.set("yze-combat", "initAutoDraw", true);
+        game.settings.set("yze-combat", "duplicateCombatantOnCombatStart", true);
+        game.settings.set("yze-combat", "actorSpeedAttribute", "system.ferocity");
+        game.settings.set("dragonbane", "configuredYzeCombat", true);
+    } catch (e) {
+        console.error("Dragonbane: Could not configure YZE Combat. Try refreshing the page");
+    }
+})
+
```

Comment:
These are the actual changes.

Again sorry for the ide formatting the hell out of white space here.